docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,406 @@
+---
+title: 2021 MUI Developer Survey results
+description: Results for our yearly developer survey, 2021 edition.
+date: 2022-02-22T00:00:00.000Z
+authors: ['danilo-leal']
+tags: ['Developer survey']
+---
+
+Keeping up with the tradition, a few months ago we opened the 2021 MUI Developer Survey, to which we received 1,589 responses, 7% more answers than last year. This time, we have almost doubled the survey size, adding more depth to each covered topic, especially opening space for more open-ended questions. We wholeheartedly appreciate everyone who contributed, you are all helping shape the future of MUI as a company, so, thank you!
+
+Just like the previous iterations, the survey was chopped into three sections: ["Your needs"](#your-needs), ["Your product"](#your-product), and ["About you"](#about-you).
+
+## Your needs
+
+### How would you feel if you could no longer use MUI?
+
+Fortunately, and similar to 2019 and 2020, over 93% of the respondents would be disappointed if they could no longer use MUI (a sum of _"Very disappointed"_ with _“Somewhat disappointed”_). A very slight 1% decrease from 2020 (94%) but still as encouraging as it can be. MUI’s team will continue to grow to keep you all even more satisfied.
+
+<img src="/static/blog/2021-survey/1.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Bar chart: 62.7% Very disappointed, 30.4% Somewhat disappointed, 6.9% Not disappointed." />
+
+<p class="blog-description">1567 out of 1589 answered</p>
+
+On the flip side though, the amount of "Very disappointed” alone fell by 10,3%, and the number of "Not disappointed" grew by 1,1%, in comparison with 2020. When choosing the last option, respondents were able to write a free-form answer to justify the choice. Based on the data, here are the main reasons we noticed and possible explanations for why these came to be:
+
+**The number of available alternatives**
+
+Lately, with every new sunrise, there's a new UI component library. Naturally, considering the amount of iteration each one does on the same problems, patterns start to become more and more established. It's common to see libraries with very similar implementations of components, styling solutions, theming, etc. It gets harder to find ways to differentiate, to find something that would make one's iteration more compelling. That's a main reason we do this survey yearly, to shed light into improvements opportunities. Now that we have uncovered a bunch of spaces to explore, and together with our steadily growing team, we hope to focus on more than one scope at once with the same level of attention to detail and quality we've strived to have.
```

Comment:
Let's whittle this down to the main point: it's tough to stand out in this niche, but we're working on it.

```suggestion
Lately, with every new sunrise, there's a new UI component library. Because they aim to solve the same problems, these libraries often arrive at solutions that are very similar to one another. The establishment of common industrywide patterns makes it that much more difficult to stand out from the crowd. We're working hard to expand outward from our core products in order to provide you with tools that our competitors can't match.
```